### DISCUSSION TITLE
Get rid of the :8804 in the URL?

### DISCUSSION THREAD

**Discussion by justamood** [plan-player-analytics/Plan#3205] [Q&A] [answered]
Is it possible to get rid of the :8804 in the URL?

Instead of looking like `stats.example.com:8804` it would look like `stats.example.com`

I'm using CloudFlare.

**Comment by AuroraLS3** [comment, marked as answer]
https://github.com/plan-player-analytics/Plan/wiki/External-Webserver-use:-Reverse-proxy
